### DISCUSSION TITLE
FEG Network Configuration

### DISCUSSION THREAD

**Discussion by fbiyohne** [magma/magma#15532] [General]
I have been trying to setup the ORC8R and FEG, MME, SPGW, HSS and PCRF. I am confused regarding the configurations for destination host, realm and etc for the GX, GY and S6A interfaces. Any guidance or reference on that?

**Comment by lucasgonze** [comment]
Your need for this calls out a need for documentation. Specifically we can document test cases as a framework for real world users.

**Reply by fbiyohne** [reply]
Ok, how do I get your support on this? sorry for the late reply

**Reply by lucasgonze** [reply]
Not complete, but a starting point:
* https://magma.github.io/magma/docs/feg/deploy_intro
* https://magma.github.io/magma/docs/feg/architecture_overview
* https://magma.github.io/magma/docs/basics/introduction.html
* Reading code magma/feg

**Reply by lucasgonze** [reply]
Can you take notes on what information you DONT find? If you can contribute a how-to, it would be a valuable use of your confusion.
